test/e2e/storage/testsuites/volume_group_snapshottable.go

```diff
@@ -198,19 +222,20 @@ func (s *VolumeGroupSnapshottableTestSuite) DefineTests(driver storageframework.
 					pvc.Spec.DataSource = &v1.TypedLocalObjectReference{
 						APIGroup: &group,
 						Kind:     "VolumeSnapshot",
-						Name:     volume.(map[string]interface{})["volumeSnapshotRef"].(map[string]interface{})["name"].(string),
+						Name:     volumeSnapshotName,
 					}
 
-					volSrc := v1.VolumeSource{
-						Ephemeral: &v1.EphemeralVolumeSource{
-							VolumeClaimTemplate: &v1.PersistentVolumeClaimTemplate{
-								Spec: pvc.Spec,
-							},
-						},
-					}
-					pvc, err := cs.CoreV1().PersistentVolumeClaims(f.Namespace.Name).Create(ctx, pvc, metav1.CreateOptions{})
+					// volSrc := v1.VolumeSource{
+					// 	Ephemeral: &v1.EphemeralVolumeSource{
+					// 		VolumeClaimTemplate: &v1.PersistentVolumeClaimTemplate{
+					// 			Spec: pvc.Spec,
+					// 		},
+					// 	},
+					// }
```

Comment:
why is this commented out? The created PVC will be empty and thus does not test snapshots

Reply:
In addition, the test passes as it is. Therefore it does not test that the snapshot was really restored. Please add a check that the provisioned PV contains expected content from the snapshot.